modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/odata/entity/v1_0/ObjectEntryEntityModel.java

```diff
@@ -146,7 +145,7 @@ else if (Objects.equals(
 					objectField.getDBType(),
 					ObjectFieldConstants.DB_TYPE_DATE)) {
 
-			return new DateEntityField(
```

Comment:
Hey @sergiojimcos, I think this change is OK because it's consistent with `dateCreated` and `dateModified`. But... why is `DateEntityField` not being sorted properly? Maybe there is a bug there that might affect other places?

Reply:
Oh it is because of the date format?

Reply:
Hey @4lejandrito! The problem is that if we want to reproduce the bug indicated in the ticket, the class `DateEntityField` is only possible filtering only using a `Date` format. However, if we change it using the `DateTimeEntityField` class, at this point the filtering process will allow date and time values. That is the reason why I have changed it